code/model/modelrender.cpp

```diff
@@ -2813,6 +2813,9 @@ void model_render_queue(model_render_params *interp, model_draw_list *scene, int
 	//*************************** draw the hull of the ship *********************************************
 	vec3d view_pos = scene->get_view_position();
 
+	if (is_using_showship) {
```

Comment:
The control flow can be improved by only calling `scene->get_view_position()` in the else case since we have unnecessary work here if `is_using_showship` is `true`.